src/test/java/TicketServiceTest.java

```diff
@@ -0,0 +1,211 @@
+import org.example.entity.Ticket;
+import org.example.entity.TicketType;
+import org.example.repository.TicketRepository;
+import org.example.service.impl.TicketServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.dao.DataIntegrityViolationException;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class TicketServiceTest {
+    @InjectMocks
+    private TicketServiceImpl ticketService;
+    @Mock
+    private TicketRepository ticketRepository;
+    private Ticket ticket;
+
+    @BeforeEach
+    public void setUp() {
+        ticket = mock(Ticket.class);
+    }
+
+    @Test
+    @DisplayName("Should save new ticket")
+    public void saveTicketPositiveTest() {
+        Mockito.when(ticketRepository.save(any(Ticket.class))).thenReturn(ticket);
+
+        ticketService.saveTicket(ticket);
+
+        verify(ticketRepository, times(1)).save(ticket);
+    }
+
+    @Test
+    @DisplayName("Should throw exception when ticket is null")
+    public void saveTicketNegativeTest() {
+        Mockito.when(ticketRepository.save(any(Ticket.class))).thenReturn(null);
+
+        Assertions.assertThrows(RuntimeException.class, () -> {
+            ticketService.saveTicket(null);
+        });
+    }
+
+    @Test
+    @DisplayName("Should throw exception when ticket is empty")
+    public void saveTicketCornerTest() {
```

Comment:
good that you add description but this method name also should be changed cause this one didn't tell nothing about test logic while it should